scripts/aggregate-translations/aggregate-translations.js

```diff
@@ -9,9 +9,9 @@ const writeI18nLoaders = require('./write-i18n-loaders');
 
 const defaultSearchPatterns = baseDirectory => ([
   path.resolve(baseDirectory, 'translations'), // root level translations
-  path.resolve(baseDirectory, 'node_modules', '**', 'translations'), // root level dependency translations
+  path.resolve(baseDirectory, 'node_modules', '*', 'translations'), // root level dependency translations
   path.resolve(baseDirectory, 'packages', 'terra-*', 'translations'), // package level translations
-  path.resolve(baseDirectory, 'packages', '**', 'node_modules', '**', 'translations'), // package level dependency translations
+  path.resolve(baseDirectory, 'packages', '*', 'node_modules', '*', 'translations'), // package level dependency translations
```

Comment:
previously this was `path.resolve(baseDirectory, 'packages', 'terra-*', 'node_modules', 'terra-*', 'translations'),`
https://github.com/cerner/terra-toolkit/pull/205/files#diff-b0cbeddb18aad238c54237791e92c1aaL14